installation/README.md

```diff
@@ -0,0 +1,87 @@
+# Installing Config Sync
+
+This document is a guide on how to install Config Sync on a kubernetes cluster.
```

Comment:
This document is a guide on how to manually install Config Sync on a Kubernetes cluster without Fleet Management. For installing Config Sync managed by fleet, please follow https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-config-sync.

Reply:
done